### ISSUE TITLE
Credo.Check.Readability.TrailingWhiteSpace false positive with accented letters #758

### ISSUE THREAD

**Issue by chess4ever** [rrrene/credo#757]
### Environment

* Credo version (`mix credo -v`):

`1.2.3`

* Erlang/Elixir version (`elixir -v`):

```
Erlang/OTP 22 [erts-10.5.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

Elixir 1.9.1 (compiled with Erlang/OTP 22)
```

* Operating system:
```
Debian GNU/Linux 9 (stretch)
```

### What were you trying to do?
Using credo with the *Credo.Check.Readability.TrailingWhiteSpace* rule enabled.

### Expected outcome
No issue found.

### Actual outcome
Complaining about trailing spaces at the end of a line which has not trailing white spaces.
